src/main/java/com/synopsys/integration/alert/workflow/startup/component/SystemMessageInitializer.java

```diff
@@ -64,6 +68,7 @@ public boolean validate() {
         logger.info("----------------------------------------");
         logger.info("Validating system configuration....");
 
+        clearOldMessages();
```

Comment:
This is a startup component that determines if the system is in a valid state or not.  On startup clear old messages and start clean.